macros/sfdc_get_formula_column_values.sql

```diff
@@ -27,7 +27,13 @@
                         end as value
             from {{ target_relation }}
             where object = '{{ join_to_table }}'    -- This filters the query to only include the formula fields referenced by the source table.
+
+            {% if added_inclusion_fields is not none %} -- If a users has designated fields_to_include then those will be the only fields included in the macro.
+                and {{ key }} in {{ added_inclusion_fields }}
+            {% else %}
                 and {{ key }} not in {{ var('sfdc_exclude_formulas',"('')") }} -- Excludes any formula fields the user may have set within the sfdc_exclude_formulas variable. Default is an empty string.
```

Comment:
also should we normalize everything to lower-case since we comparing against user-configured variables?

Reply:
Definitely! I'll update this

Reply:
Updated